### PR TITLE
Add VAT to contact object

### PR DESCRIPTION
**Pull request by VladimirBerdnik**
## Summary
This PR will add a VAT property to the `ContactObject` schema

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Contact/ContactObject.yaml [hunk 1]
```diff
@@ -91,3 +91,11 @@ properties:
     maxLength: 40
     example: 056ae6d97c788b9e98b049ebafd7b229bf852221
     readOnly: true
+  vat:
+    description: |-
+      A value-added tax identification number. Valid VAT number that passes a check
+      at the EU's official [VIES website](https://ec.europa.eu/taxation_customs/vies/#/vat-validation).
```
Comment (adamaltman) [inline openapi/components/schemas/Contact/ContactObject.yaml:97]: https://sembr.org/

**Review by adamaltman — DISMISSED** [review]

**Comment by ashkarpetin** [comment]
No Writing style and API design standards checks?

File: openapi/components/schemas/Contact/ContactObject.yaml [hunk 2]
```diff
@@ -91,3 +91,11 @@ properties:
     maxLength: 40
     example: 056ae6d97c788b9e98b049ebafd7b229bf852221
     readOnly: true
+  vat:
+    description: |-
+      A value-added tax identification number.
```
Comment (adamaltman) [inline openapi/components/schemas/Contact/ContactObject.yaml:96]:
```suggestion
      An EU value-added tax identification number.
```

File: openapi/components/schemas/Contact/ContactObject.yaml [hunk 3]
```diff
@@ -91,3 +91,11 @@ properties:
...
+      Valid VAT number that passes a check at the EU's official [VIES website](https://ec.europa.eu/taxation_customs/vies/#/vat-validation).
```
Comment (adamaltman) [inline openapi/components/schemas/Contact/ContactObject.yaml:97]:
```suggestion
      It must pass the official EU official [VIES validation](https://ec.europa.eu/taxation_customs/vies/#/vat-validation).
```

File: openapi/components/schemas/Contact/ContactObject.yaml [hunk 4]
```diff
@@ -91,3 +91,11 @@ properties:
     maxLength: 40
     example: 056ae6d97c788b9e98b049ebafd7b229bf852221
     readOnly: true
+  vat:
+    description: |-
+      An EU value-added tax identification number.
```
Comment (adamaltman) [inline openapi/components/schemas/Contact/ContactObject.yaml:96]:
```suggestion
      EU value-added tax identification number.
```

**Review by ashkarpetin — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Comment by subenson** [comment]
I propose to change `vat` to `vatNumber`. A contact does not have tax, it has a tax identification number.

**Pull request closed without merging**